ui/src/components/Settings/About/index.tsx

```diff
@@ -0,0 +1,322 @@
+import { InformationCircleIcon } from '@heroicons/react/solid'
+import React, { useContext, useEffect, useRef, useState } from 'react'
+import GetApiHandler from '../../../utils/ApiHandler'
+import Releases from './Releases'
+
+interface VersionResponse {
+  status: 1 | 0
+  version: string
+  commitTag: string
+  updateAvailable: boolean
+}
+interface CollectionsResponse {
+  id: number
+  plexId: number
+  libraryId: number
+  title: string
+  description: string
+  isActive: boolean
+  arrAction: number
+  visibleOnHome: boolean
+  deleteAfterDays: number
+  manualCollection: boolean
+  manualCollectionName: string
+  listExclusions: boolean
+  forceOverseerr: boolean
+  type: string
+  keepLogsForMonths: number
+  addDate: Date
+  handledMediaAmount: number
+  lastDurationInSeconds: number
+  tautulliWatchedPercentOverride: string
+  radarrSettingsId: number
+  sonarrSettingsId: number
+  media: string[]
+}
+
+const AboutSettings = () => {
+  useEffect(() => {
+    document.title = 'Maintainerr - Settings - About'
+  }, [])
+  // Timezone
+  const getBrowserTimezone = (): string => {
+    return Intl.DateTimeFormat().resolvedOptions().timeZone
+  }
+  // End Timezone
+  // Maintainerr Version
+  const [version, setVersion] = useState<string>('')
+  const [commitTag, setCommitTag] = useState<string>('')
+
+  useEffect(() => {
+    GetApiHandler('/app/status').then((resp: VersionResponse) => {
+      if (resp.status) {
+        setVersion(resp.version)
+        setCommitTag(resp.commitTag)
+      }
+    })
+  }, [])
+  // End Maintainerr Version
+  // Maintainerr Rules Count
+  const [ruleCount, setRuleCount] = useState<number | null>(null) // To store the count
+  useEffect(() => {
+    GetApiHandler('/rules').then((resp) => {
+      if (resp) {
+        let totalCount = 0
+        if (Array.isArray(resp)) {
+          totalCount = resp.length
+        }
+        setRuleCount(totalCount)
+      }
+    })
+  })
+  // End Maintainerr Rules Count
+  // Maintainerr Collection Items
+  const [itemCount, setItemCount] = useState<number | null>(null) // To store the count
+  useEffect(() => {
+    GetApiHandler('/collections/media').then((resp) => {
```

Comment:
You would be much better off introducing a new endpoint like `/collections/media/count` which returns only the count instead of bringing back all the data just to count it.

Reply:
Wouldn't it end up doing the same thing? Just on the backend? I don't know, but I assume the endpoint would have to do the same thing and then return the count?

Reply:
No, you would use a COUNT SQL query, with typeorm it would look something like this:
```
const count = await this.CollectionMediaRepo.count({
  where: { collectionId: id },
})```